### PR TITLE
HEEDLS-216 Section page basic info front end

### PR DESCRIPTION
**Pull request by kwunkiuma**
Adds basic info to section page.

Displays the section title, %complete, minutes spent, average minutes spent, and back button.

image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 1]
```diff
@@ -3,6 +3,30 @@
 
 <link rel="stylesheet" href="@Url.Content("~/css/learningMenu/index.css")" asp-append-version="true">
 
+@{
+  ViewData["HeaderPrefix"] = "";
+  ViewData["Application"] = "Model.Title";
+  ViewData["Title"] = "Learning Menu";
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:9]: As commented on my PR, this should be set to the course title 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 2]
```diff
@@ -3,6 +3,30 @@
 
 <link rel="stylesheet" href="@Url.Content("~/css/learningMenu/index.css")" asp-append-version="true">
 
+@{
+  ViewData["HeaderPrefix"] = "";
+  ViewData["Application"] = "Model.Title";
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:8]: Better to hardcode this to an actual value 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 3]
```diff
@@ -3,6 +3,30 @@
...
+  ViewData["CustomisationId"] = 18438;
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/18438";
+  ViewData["HeaderPathName"] = "Model.Title";
+}
 @section NavMenuItems {
   <partial name="Shared/_NavMenuItems" />
 }
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading" class="nhsuk-heading-xl">Model.SectionTitle</h1>
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:19]: Better to hardcode this to an actual value 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Comment by kwunkiuma** [comment]
Here is what the page now looks like:
image

**Pull request merged**